### ISSUE TITLE
clang warning shows a bug

### ISSUE THREAD

**Issue by marcespie** [LinuxBeaver/GEGL-Custom-Bevel#1]
Clang warns on line 251 of the plugin, because the else is incorrectly indented
I would assume the "else" branch for restorepuff should encompass both statements

**Comment by LinuxBeaver** [comment]
Yeah, Restorepuff is enabled by default. When it is disabled Custom Bevel and Glossy Balloon run a command that stack multiple duplicate layers of their output on top of each other until all transparency goes away. I am aware the "else if" area is indented incorrectly and I will work on fixing it.
